### ISSUE TITLE
SubSuites for skiped tests

### ISSUE THREAD

**Issue by yannay000** [Shelex/cypress-allure-plugin#58]
**Describe the bug**
Tests with label _subSuite_ and with mark _skip_ lose their suites.

**Example**

describe('suiteName', function(){

- cy.allure().label('subSuite', 'subSuiteName');

-  it.skip('testName', function () {

- .......

-  }

}

**Current behavior**
Test _testName_ will be situated in folder _suiteName_.

**Desired behavior**
Test _testName_ should be situated in subfolder _subSuiteName_ in folder _suiteName_.

**Comment by maga-polito** [comment]
Having the same issue using cucumber

**Comment by Shelex** [comment]
@yannay000 @maga-polito 

Thank you for information provided. The source of issue is that labels like suite/subsuite/parentsuite and others could be applied to allure test only, so I have published v2.4.0 where there is new label storage. When there is no test plugin now gather these labels and applies them to each test in that suite (describe block).
```
describe('label storage', () => {
    const allure = Cypress.Allure.reporter.getInterface();
    allure.label('subSuite', 'global sub suite');

    it.skip('skipped should be in global sub suite', () => {
        allure.label('subSuite', 'local sub suite from skipped');
        expect(true).to.be.eq(true);
    });

    it('should be in global sub suite', () => {
        expect(true).to.be.eq(true);
    });

    it('should be in local sub suite', () => {
        allure.label('subSuite', 'local sub suite');
        expect(true).to.be.eq(true);
    });
});
```

<img width="625" alt="labelStorageReport" src="https://user-images.githubusercontent.com/11396724/113181951-021bfd00-925b-11eb-8cde-f07f25c49a3c.png">
